response_operations_ui/forms.py

```diff
@@ -242,3 +242,38 @@ def validate_password(form, field):
         if password.isalnum() or not any(char.isupper() for char in password) or not any(char.isdigit() for char in
                                                                                          password):
             raise ValidationError('Your password doesn\'t meet the requirements')
+
+
+class RequestAccountForm(FlaskForm):
```

Comment:
This should have a docstring saying what it's used for.  Took me a look around the codebase to figure that out